public/trips.js

```diff
@@ -1,73 +1,75 @@
+import { uid } from "uid";
+
 const trips = [
   {
     location: "Greece",
-    startDate: "12/09/23",
-    endDate: "02/10/23",
+    startDate: "09/12/23",
+    endDate: "10/02/23",
     image: "/greece.jpg",
-    id: "001",
+    id: uid(),
   },
```

Comment:
please don't put this file into the public folder. This should only contain elements that are shipped directly to the client. You can create a db folder and put this file in there.